### ISSUE TITLE
Experimental setup

### ISSUE THREAD

**Issue by sl403** [pariajm/deep-disfluency-detection#2]
Hi, 

What's your experimental setup? 
Is python2? What's your tensorflow version? 
thank you.

**Comment by pariajm** [comment]
> Hi,
> 
> What's your experimental setup?
> Is python2? What's your tensorflow version?
> thank you.

Hi, 
It should work with python 3 and tensorflow > 0.12.
I updated the repo (fixed print commands).

**Comment by sl403** [comment]
Thank for your reply.

Another question unrelated to the title.
Can you release some trained model and the predict code make me to predict it?
If possible, can you provide your experimental test data?
Thank you.

**Comment by pariajm** [comment]
> Thank for your reply.
> 
> Another question unrelated to the title.
> Can you release some trained model and the predict code make me to predict it?
> If possible, can you provide your experimental test data?
> Thank you.

Since Switchboard is not open-source, I cannot release the test set (you can follow the pre-processing steps and use the exact test set split explained in the paper though). 
But I just released the trained model and the prediction code. Check the repo, please!
